src/main/java/com/sleekbyte/tailor/utils/Configuration.java

```diff
@@ -127,6 +134,9 @@ public boolean debugFlagSet() throws CLIArgumentParserException {
 
         if (pathNames.size() >= 1) {
             fileNames.addAll(findFilesInPaths(pathNames));
+        } else if (Files.isReadable(CODE_CLIMATE_CONFIG)) {
+            pathNames.addAll(getCodeClimateIncludePaths());
+            fileNames.addAll(findFilesInPaths(pathNames));
```

Comment:
So the user won't be able to use the config file if they are analyzing via code climate?